ormconfig.js

```diff
@@ -0,0 +1,31 @@
+const dotenv = require('dotenv');
+
+const envFile = process.env.ENVIRONMENT
+  ? `.env.${process.env.ENVIRONMENT}`
+  : '.env';
+dotenv.config({ path: envFile });
+
+
+// If .env wasn't provided then exit
+if (!process.env.PORT) {
+  console.error('==> Please check your .env');
+  process.exit(1);
+}
+
+module.exports = {
+  type: 'postgres',
+  host: process.env.TYPEORM_HOST,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+  port: Number.parseInt(process.env.TYPEORM_PORT || '3000'),
+  synchronize: process.env.TYPEORM_SYNCHRONIZE === 'true',
+  logging: process.env.TYPEORM_LOGGING === 'true',
```

Comment:
Maybe here we can use only the environment variable because this field can fill up with `true`, `false` or an array with the different types of logging(e.g `["query", "error"]`), WDYT?

Reply:
Also, maybe we can create a `.example.env` to document all the environment variables in one file